DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -26,6 +31,12 @@ public interface IEventsSheetsService
 
         Task<DiscordEvent> GetEventAsync(int eventKey);
         Task<IEnumerable<DiscordEvent>> ListEventsAsync();
+
+        Task AddResponseOfUser(int eventKey, ulong userId, string responseEmoji);
+        Task ClearResponsesOfUser(int eventKey, ulong userId);
```

Comment:
Maybe `ForUser` rather than `OfUser`? 🙂

Reply:
Yeah that's clearer 🙂 Changed in 1718faa